homeassistant/components/min_max/sensor.py

```diff
@@ -253,7 +257,9 @@ async def async_added_to_hass(self) -> None:
         # Replay current state of source entities
         for entity_id in self._entity_ids:
             state = self.hass.states.get(entity_id)
-            state_event = Event("", {"entity_id": entity_id, "new_state": state})
+            state_event: EventType[EventStateChangedData] = EventType(
```

Comment:
Was this intended?
```suggestion
            state_event: EventType[EventStateChangedData] = Event(
```

Reply:
Yes. `state_event` is passed to the (reused) callback function, thus requiring the type to be `EventType[...]`. Additionally, `EventType` is a subclass of `Event` so the change is necessary (for now).